src/js/stage.js

```diff
@@ -213,8 +213,11 @@ define(["d3", "palette", "transitions/default"], function (d3, palette, defaults
           .data(function (d) {
             return d.labels;
           });
-        labels.enter().append("text");
         transitions.labels(labels, labelAttributer);
+        labels.enter()
+          .append("text")
+          .call(labelAttributer)
+          .call(styliseur);
```

Comment:
I think that it is not needed only for d3.js version switch but it appears to fix a bug. Am I right?